### PR TITLE
Moved shared specs to includable location

### PR DESCRIPTION
**Pull request by afred**

### CODE REVIEW

**Comment by cjcolvar** [comment]
@afred I moved the shared spec and renamed it based upon what I saw in valkyrie and hyrax-iiif_av.

**Comment by afred** [comment]
@tpendragon @cjcolvar I took a stab at adding an example in the README, but please look over my comments and make sure they're accurate. Still trying to figure it out myself actually.

File: README.md [hunk 1]
```diff
@@ -124,6 +124,75 @@ end
...
+    end
+  end
+end
+```
+Then, use the shared specs...
+```ruby
+# In your application at...
+# spec/lib/active_encode/engine_adapters/my_custom_adapter_spec.rb
+require 'spec_helper'
+require 'active_encode/shared_specs/engine_adapter_specs'
```
Comment (cjcolvar) [inline README.md]: With the moving around that I did I think this line should now be:
```suggestion
require 'active_encode/spec/shared_specs'
```

**Comment by cjcolvar** [comment]
@afred I think that looks really good!

**Pull request merged**